zebra-network/src/peer_set/initialize.rs

```diff
@@ -29,7 +29,7 @@ use zebra_chain::{chain_tip::ChainTip, parameters::Network};
 
 use crate::{
     constants,
-    meta_addr::MetaAddr,
+    meta_addr::{MetaAddr, MetaAddrChange},
     peer::{self, HandshakeRequest, OutboundConnectorRequest},
     peer_set::{set::MorePeers, ActiveConnectionCounter, CandidateSet, ConnectionTracker, PeerSet},
     timestamp_collector::TimestampCollector,
```

Comment:
Maybe I should rename the `TimestampCollector` `AddressBookUpdater`? The reason is that the `TimestampCollector` is not used only to collect timestamps but also to add the unused peers to the `AddressBook`. I can't think of a better name, though.

Reply:
Please make sure you rename the timestamp collector type, module, and variables.
(And you're right, the timestamp collector was never only about timestamps.)